exercises/practice/beer-song/.meta/config.json

```diff
@@ -1,10 +1,10 @@
 {
   "blurb": "Recite the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.",
   "authors": [
-    "FIXME-marnen"
+    "russtopia"
   ],
   "contributors": [
-    "russtopia"
+    "ErikSchierboom"
```

Comment:
I've hardly done anything :)

```suggestion
```

A contributor should provide a substantial contribution, which I don't think I have :)

Reply:
:) :+1: